nifi-nar-bundles/nifi-cybersecurity-bundle/nifi-cybersecurity-processors/src/main/java/org/apache/nifi/processors/cybersecurity/FuzzyHashContent.java

```diff
@@ -73,31 +70,14 @@
         "evaluations in memory. Accordingly, it is important to consider the anticipated profile of content being " +
         "evaluated by this processor and the hardware supporting it especially when working against large files.")
 
-@SeeAlso({HashContent.class})
+@SeeAlso({HashContent.class, CompareFuzzyHash.class})
```

Comment:
This refers to a processor in another NAR, which won't be able to be found by the framework. I'll remove the reference to HashContent (and the standard processors dependency in the POM) on merge